docs/pages/docs/ui-components/suggestion-menus.mdx

```diff
@@ -31,20 +31,56 @@ Passing `slashMenu={false}` to `BlockNoteView` tells BlockNote not to show the d
 
 `getItems` should return the items that need to be shown in the Slash Menu, based on a `query` entered by the user (anything the user types after the `triggerCharacter`).
 
-### Replacing the Suggestion Menu Component
+### Replacing the Slash Menu Component
```

Comment:
I'm afraid extending this doc with grid + emoji will add to many conepts in one

Reply:
I think it's ok tbh, but maybe better to have a separate page for grid suggestion menus and leave the current suggestion menu page as-is?

Reply:
Imagine you're new to blocknote and open this page. Not too much information to grasp all concepts? Besides, I think the different UI pages now follow the same "flow" making it easier to grasp, I think we're also breaking that with this change?

Reply:
Hmm ok, so do you think the suggestion of splitting the grid suggestion menu w/ emoji picker and regular suggestion menu w/ slash menu makes sense? That would keep the structure consistent and have less information on a single page.

Reply:
- Yes, my suggestion would be to move the emoji and grid suggestionmenu info to a single page under Advanced. I think customizing these components is pretty niche.
- The emojiPicker needs to be added to the props here: https://blocknote-website-1u5ylvcga-typecell.vercel.app/docs/editor-basics/setup#rendering-the-editor-with-blocknoteview